docs/product/performance/cache-monitoring/cache-page.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "Cache Page"
+sidebar_order: 55
+description: "Learn more about Sentry's Cache page where you can get an overview of your cache performance and catch potential cache performance issues. "
+---
+
+The **Caches** page gives an overview of cache performance across all endpoints for currently selected backend projects with summary graphs for **Miss Rate** and **Requests Per Minute** (throughput). You can use these as a starting point to see if there are any potential cache performance issues, for example, a higher than expected Miss Rate percentage.
+
+ If you see an anomaly or want to investigate a time range further, click and drag to select a range directly in the graph and you'll see data for that specific time range only.
+
+The transaction table shows a list of endpoints that contain at least one `cache.get` span along with:
+
+- Its average value size (the bytes being fetched from cache)
+- Throughput
+- Average transaction duration
+- Miss rate percentage (how often did a lookup from cache fail)
```

Comment:
```suggestion
- Miss rate percentage (how often a cache lookup failed)
```

Reply:
Its not really a failed lookup, its a lookup that returned no value
```suggestion
- Miss rate percentage (how often did a lookup did not return a value)
```